apps/api/src/configuration/configuration.schema.ts

```diff
@@ -3,6 +3,7 @@ import { z } from 'zod';
 
 export const $Configuration = z.object({
   API_DEV_SERVER_PORT: z.coerce.number().positive().int().optional(),
+  GATEWAY_API_KEY: z.string().min(32),
```

Comment:
If you require a minimum size, say so in the env template.

Reply:
bb9e463fe44bcb7c9f09b3f1c0be9fa960d507d2